src/scene/mesh.js

```diff
@@ -981,6 +997,85 @@ class Mesh extends RefCountedObject {
         };
         this.indexBuffer[RENDERSTYLE_WIREFRAME] = wireBuffer;
     }
+
+    /**
+     * Builds the array of triangles of the mesh
+     */
```

Comment:
functions that are not public need to be marked with `@ignore`

Reply:
Done